holidays/financial/japan_exchange.py

```
@@ -0,0 +1,64 @@
+
+        # Jan 2, Jan 3, Dec 31
+        self._populate_bank_holidays()
+
+    def _populate_bank_holidays(self):
+        name = tr("市場休業日")
+        self._add_new_years_day_two(name)
+        self._add_new_years_day_three(name)
+        self._add_new_years_eve(name)
+
+
+# Exchange aliases – all refer to the same JapanExchange calendar.
```

Ruff (0.15.1)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)